src/Datadog.Trace/DogStatsD/TracerMetricNames.cs

```diff
@@ -0,0 +1,73 @@
+namespace Datadog.Trace.DogStatsD
+{
+    internal static class TracerMetricNames
+    {
+        public static class Api
+        {
+            /// <summary>
+            /// Count: Total number of API requests made
+            /// </summary>
+            public const string Requests = "datadog.tracer.api.requests";
+
+            /// <summary>
+            /// Count: Count of API responses.
+            /// This metric has an additional tag of "status: {code}" to group the responses by the HTTP response code.
+            /// This is different from <seealso cref="Errors"/> in that this is all HTTP responses
+            /// regardless of status code, and <seealso cref="Errors"/> is exceptions raised from making an API call.
+            /// </summary>
+            public const string ResponsesByStatusCode = "datadog.tracer.api.responses";
+
+            /// <summary>
+            /// Count: Total number of exceptions raised by API calls.
+            /// This is different from receiving a 4xx or 5xx response.
+            /// It is a "timeout error" or something from making the API call.
+            /// </summary>
+            public const string Errors = "datadog.tracer.api.errors";
+        }
+
+        public static class Queue
+        {
+            /// <summary>
+            /// Count: Total number of traces pushed into the queue, where "accepted - dropped = total to be flushed"
+            /// </summary>
+            public const string EnqueuedTraces = "datadog.tracer.queue.accepted";
+
+            /// <summary>
+            /// Count: Total number of spans pushed into the queue
+            /// </summary>
+            public const string EnqueuedSpans = "datadog.tracer.queue.accepted_lengths";
+
+            /// <summary>
+            /// Count: Total size in bytes of traces pushed into the queue
+            /// </summary>
+            public const string EnqueuedBytes = "datadog.tracer.queue.accepted_size";
```

Comment:
Are these standardized names?
Otherwise, setting this up for future use?

Reply:
Yep!